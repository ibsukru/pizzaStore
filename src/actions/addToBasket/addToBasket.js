const addBasket = (pizza) => {
  return {
    payload: pizza,
    type: 'BASKET_ADD'
  }
}

export default addBasket
