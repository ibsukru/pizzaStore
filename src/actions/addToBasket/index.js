import addToBasket from './addToBasket'

export default addToBasket
