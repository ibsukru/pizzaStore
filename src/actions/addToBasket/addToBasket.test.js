/* eslint-env jest */

import addToBasket from './addToBasket'

[
  {},
  {
    pizza: {
      name: 'large'
    }
  }
].forEach(payload =>
  test(`Invokes action for payload ${JSON.stringify(payload)}`, () => {
    expect(addToBasket(payload)).toMatchSnapshot()
  })
)
