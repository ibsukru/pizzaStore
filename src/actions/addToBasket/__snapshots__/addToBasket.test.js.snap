// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Invokes action for payload {"pizza":{"name":"large"}} 1`] = `
Object {
  "payload": Object {
    "pizza": Object {
      "name": "large",
    },
  },
  "type": "BASKET_ADD",
}
`;

exports[`Invokes action for payload {} 1`] = `
Object {
  "payload": Object {},
  "type": "BASKET_ADD",
}
`;
